xdsl/tools/tblgen_to_py.py

```diff
@@ -629,3 +631,7 @@ def main():
             tblgen_to_py(args.input_file, output, cull=args.cull)
     else:
         tblgen_to_py(args.input_file, cull=args.cull)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Just so you know, you can run this script without this change with `xdsl-tblgen` with your virtualenv activated (or `uv run xdsl-tblgen`)

Reply:
ah right forgot about that :) it was a bit simpler for me to debug calling main directly but I guess it is not needed in this PR